src/Keboola/DebugLogUploader/UploaderAbs.php

```diff
@@ -12,18 +12,13 @@ class UploaderAbs implements UploaderInterface
     public const PAD_LENGTH = 5;
     public const CHUNK_SIZE = 4 * 1024 * 1024;
 
-    /** @var string */
-    private $urlPrefix;
+    private string $urlPrefix;
 
-    /** @var string */
-    private $container;
+    private string $container;
 
-    /** @var string */
-    private $uploadPath;
+    private string $uploadPath;
 
-    private \MicrosoftAzure\Storage\Blob\BlobRestProxy $absClient;
-
-    public function __construct(BlobRestProxy $absClient, array $config)
+    public function __construct(private readonly BlobRestProxy $client, array $config)
```

Comment:
Hlasoval bych za to, že promoted property musí mít vlastní řádek... ale tady už to nech...

Reply:
Nenašel jsem vhodný sniff :( (jen v phpcsfixeru :( )